strr-platform-web/app/components/form/platform/BusinessDetails.vue

```diff
@@ -117,45 +87,50 @@ watch(canadaPostAddress, (newAddress) => {
   <div data-testid="business-details">
     <ConnectPageSection
       class="bg-white"
-      :heading="{ label: tPlat('section.title.businessInfo'), labelClass: 'font-bold md:ml-6' }"
+      :heading="{ label: $t('platform.section.title.businessInfo'), labelClass: 'font-bold md:ml-6' }"
     >
       <UForm
-        ref="platformBusinessRef"
-        :schema="businessdetailsSchema"
+        ref="platformBusinessFormRef"
+        :schema="getBusinessSchema()"
```

Comment:
It seems like the reactivity here or below in the sections where its calling 'hasFormErrors' is not working as expected. When the user changes if they have a cpbc number or registered office the section validation styling doesn't update anymore.

This also may have to do with why the step validation isn't working for this one

Reply:
Ah dang, good catch